jetbrains-core/src/software/aws/toolkits/jetbrains/services/ecs/exec/EcsExecUtils.kt

```diff
@@ -212,7 +212,8 @@ object EcsExecUtils {
             it.cluster(service.clusterArn())
             it.services(service.serviceArn())
         }
-        val serviceStateChangeInProgress = response.services().first().deployments().first().rolloutState() == DeploymentRolloutState.IN_PROGRESS
+        val deployment = response.services().first().deployments().first()
+        val serviceStateChangeInProgress = deployment.rolloutState() == DeploymentRolloutState.IN_PROGRESS || deployment.status() == "ACTIVE"
```

Comment:
I'm shocked this isn't an Enum in the model - weird.

Reply:
yep, it's a string....